### PR TITLE
Reports api

### PR DESCRIPTION
**Pull request by maykremer**

### CODE REVIEW

File: GetReports/index.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    try {
+        if (
+            user.role !== Role.RamadIturOfUnit &&
+            user.role !== Role.RamadIturAssistant &&
+            user.role !== Role.ProfessionalRamad &&
+            user.role !== Role.Mada &&
+            user.role !== Role.Itur &&
+            user.role !== Role.Psychologist &&
+            user.role !== Role.Diagnoser &&
+            user.role !== Role.Technical
```
Comment (carmeltsadok) [inline GetReports/index.ts:27]: Change if not includes in array of roles

File: GetReports/index.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        const validationError: ValidationError = getReportsSchema.validate(req).error;
+        if (validationError) {
+            throw new ValidationFunctionError(functionName, validationError.message);
+        }
+
+        const { startDate, endDate } = req.body;
+
+        const events: Event[] = await EventModel.find({ time: { $gte: startDate, $lte: endDate } }).exec();
+
+        if (events.length < 1) {
```
Comment (carmeltsadok) [inline GetReports/index.ts:43]:
```suggestion
        if (!events.length) {

```

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

File: GetReports/function.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["get"],
+      "route": "reports"
+
+    },
```
Comment (lustigy96) [inline GetReports/function.json:11]:
```suggestion
      "route": "reports"
    },
```

File: GetReports/index.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                config.fields.malshabLastName,
+                config.fields.malshabFirstName,
+                config.fields.malshabId,
+                config.fields.nodeId,
+                config.fields.nodeName,
+            ],
+        });
+
+        return {
+            status: 202,
```
Comment (lustigy96) [inline GetReports/index.ts:79]: 303 status code is 'accepted' It is irrelevant to a get function
```suggestion

```

File: GetReports/config.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+export default {
```
Comment (lustigy96) [inline GetReports/config.ts:1]: Where is the characterization?

**Review by lustigy96 — CHANGES_REQUESTED** [review]

**Review by carmeltsadok — APPROVED** [review]

**Review by lustigy96 — APPROVED** [review]

**Review by carmeltsadok — APPROVED** [review]

**Pull request merged**